architecture/core_concepts/routes.adoc

```diff
@@ -802,3 +802,34 @@ If unit not provided, ms is the default.
 Setting a server-side timeout value for passthrough routes too low can cause
 WebSocket connections to timeout frequently on that route.
 ====
+
+
+[[wildcard-subdomain-route-policy]]
+== Creating routes specifying a wildcard subdomain policy
+
+A route can specify a wildcard policy as part of its configuration using the
```

Comment:
Can we say what they do as a first sentence?

"A wildcard policy allows a user to define a route that can cover all hosts within a domain (when the router is configured to allow it)."

Or something